medcat/cat.py

```diff
@@ -390,12 +390,7 @@ def load_model_pack(cls,
         model_pack_path = cls.attempt_unpack(zip_path)
 
         # Load the CDB
-        cdb_path = os.path.join(model_pack_path, "cdb.dat")
-        nr_of_jsons_expected = len(SPECIALITY_NAMES) - len(ONE2MANY)
-        has_jsons = len(glob.glob(os.path.join(model_pack_path, '*.json'))) >= nr_of_jsons_expected
-        json_path = model_pack_path if has_jsons else None
-        logger.info('Loading model pack with %s', 'JSON format' if json_path else 'dill format')
-        cdb = CDB.load(cdb_path, json_path)
+        cdb = cls.load_cdb(model_pack_path)
```

Comment:
I don't really know why the local imports are here (or above in the method at least). My best guess is that at some point this helped avoid circular imports while at the same time provided a guarantee that the classes would be available during model pack loading. But currently, this shouldn't really be an issue since all 4 are imported at the top of the module anyway so there's no reason they wouldn't be available during operation.

So I don't see why we wouldn't just remove the local imports in here.

With that said, that would probably fall outside the scope of this PR.

So I think the easiest solution to the linting issue would be to type-annotate here, e.g `cdb: CDB = cls.load_cdb(model_pack_path)`.

Reply:
done - yeah on above local imports. Agreed prob due to circular imports somewhere... and yes outside of scope of this PR